src/VideoClip.js

```diff
@@ -2,49 +2,44 @@ const MC = require("@kissmybutton/motorcortex");
 
 class VideoClip extends MC.BrowserClip {
   get html() {
+    this.width = this.attrs.width || 640;
+    this.height = this.attrs.height || 360;
+    this.startFrom = this.attrs.startFrom || 0;
+
+    const videoStyle = `width:${this.width}px;height:${this.height}px;`;
+    const videoSources = this.attrs.sources
+      .map((item) => `<source src="${item}#t=${this.startFrom}"></source>`)
+      .join("\n");
+
     return `
-        <div>
-            <video id="video" style="width:${
-              this.attrs.width || 640
-            }px;height:${this.attrs.height || 360}px;" preload="auto">
-                ${this.attrs.sources
-                  .map(
-                    (item /*, i*/) => `
-                    <source src="${item}#t=${
-                      this.attrs.startFrom || 0
-                    }"></source>
-                `
-                  )
-                  .join("")}
-            </video>
-            <canvas id="canvas"></canvas>
-        </div>
-        `;
+      <div>
+          <video id="video" style="${videoStyle}" preload="auto">
+              ${videoSources}
+          </video>
+          <canvas id="canvas"></canvas>
+      </div>
+    `;
   }
 
   get css() {
```

Comment:
can't we avoid the getter here and just use a static value?

Reply:
I think not. We should ask trad

Reply:
For the consumer of this (MC in this case) there shouldn't be any difference between a getter that has a static value and just assigning that value. Meaning the following two examples should be the same
```js
class Smth{
  get css(){ return 'css1'}
}
```

```js
class Smth{
  css = 'css1'
}
```

@andreas-trad any thoughts?